client/src/components/Common/Logo/index.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styled from "styled-components";
+
+const Wrapper = styled.div`
+  height: 100%;
+  width: 100%;
+`;
+
+const Img = styled.img`
+  outline: none;
+  user-drag: none;
+  user-select: none;
+`;
+
+const LogoPaths = {
+  'mark-only': 'logo_markonly.svg',
+  'mark-with-text': 'logo_marktext.svg',
+};
+
+const Logo = (props) => {
```

Comment:
Instead of generic props, we can destructure props and specifically pull out the variable(s) needed. 
You can update (props) to be ({type}).

Reply:
oooh heck yeah! that's what I was looking for! Thanks 🚀